src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -500,8 +508,9 @@ const uploadItems = async (props: DriveExplorerProps, rootList: IRoot[], files:
         }),
       );
     }
-    if (rootList.length) {
-      for (const root of rootList) {
+
+    if (foldersUploadedNotExist.length) {
```

Comment:
why the length is needed to be checked?

Reply:
To check that it has elements, if there are no elements there is no need to execute the for